include/picongpu/fields/currentDeposition/EmZ/DepositCurrent.hpp

```diff
@@ -35,8 +35,8 @@ namespace picongpu
         {
             using namespace pmacc;
 
-            template<typename T_AtomicAddOp, typename ParticleAssign, int T_begin, int T_end>
-            struct DepositCurrent<T_AtomicAddOp, ParticleAssign, T_begin, T_end, DIM3> : public Base<ParticleAssign>
+            template<typename T_Strategy, typename ParticleAssign, int T_begin, int T_end>
```

Comment:
Unrelated to this PR, possibly for a follow-up you are planning anyways: `ParticleAssign` should be `T_ParticleAssign`